homeassistant/components/smartenergy_goecharger/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "smartenergy_goecharger",
+  "name": "go-e Charger Cloud",
+  "documentation": "https://www.home-assistant.io/integrations/smartenergy_goecharger",
+  "iot_class": "cloud_polling",
+  "config_flow": true,
+  "dependencies": ["sensor"],
```

Comment:
I don't know if you need to have sensor as a dependency, I've never seen that before, but I'm not sure.